.clj-kondo/config.edn

```diff
@@ -594,7 +598,6 @@
     metabase.api.timeline-event                                   api.timeline-event
     metabase.api.user                                             api.user
     metabase.api.util                                             api.util
-    metabase.async.streaming-response.thread-pool                 thread-pool
```

Comment:
now that we're modularizing stuff telling everyone they have to refer to some very specific internal namespace in a module by a specific alias doesn't make sense anymore